MonoGame.Framework.Content.Pipeline/Builder/PipelineBuildEvent.cs

```diff
@@ -180,13 +180,19 @@ public bool NeedsRebuild(PipelineBuildEvent cachedEvent)
                 cachedEvent.DestFile != DestFile)
                 return true;
 
+            // Did the importer assembly change?
+            if (manager.GetImporterAssemblyTimestamp(cachedEvent.Importer) > cachedEvent.DestTime)
```

Comment:
I hadn't thought of that scenario, but yes, using `DestTime` did feel a bit brittle. I'll put together another pull request with that change.